feat: 6167 Add autoplay setting in the admin settings section

Khyvodul opened the pull request:
## Description

Add section in the admin's GUI to enable or disable the video autoplay for new or anonymous users

The autoplay feature is kept enabled by default to ensure consistency when upgrading from previous versions.

As already existing, user can still set autoplay on or off in its own preferences, which will override the instance's value.

## Related issues

Allow instance admin to set autoplay off by default
https://github.com/Chocobozzz/PeerTube/issues/6167


## Has this been tested?

- [x] 👍 yes, I added tests to the test suite

## Screenshots

![screen_capture](https://github.com/user-attachments/assets/b71eb443-178a-4de2-a726-92686ee0a303)

Chocobozzz commented:
HI and thanks! Please revert your changes in `clietn/src/locale`, they are automatically generated in `develop` branch :)

Khyvodul commented:
Hi,
I was wondering if I should include these changes or not (hence the separate commit just for them), now I have my answer :-).
I've reverted the commit and removed all changes to _client/src/locale/_.

Please let me know if more changes are needed.
Thanks

Chocobozzz commented:
Thanks!

I just moved the setting to the `defaults` section in the config. Since this section cannot be updated in the admin config (yet), I removed this part from the PR.

A nice addition in the future would to add all `defaults` config in the admin client :)
